fms/modules/attention.py

```diff
@@ -591,6 +591,7 @@ def __init__(
         position_encoder: Optional[PositionEncoder] = None,
         fused: bool = True,
         group: Optional[ProcessGroup] = None,
+        use_fp8_kvcache: bool = False,
```

Comment:
there may be other kinds of quantization we'll want to add later, and we won't want to have a long list of optional booleans to specify which kind of quantization to use. could this be more generic? maybe instead of boolean this could be some kind of dtype parameter?

Reply:
In other open-source implementations, quantization config is typically more than just a boolean flag or a dtype parameter. There are combinations of things that can be used. Right now, this enables testing MHA in TP and doesn't do more than that and this is the method Antoni uses in non-TP MHA.

Reply:
I made some suggestions on #313 on how to handle this. In face, #313 has exactly the same issue with GPTQ